### PR TITLE
basic docker-commands

### PR DESCRIPTION
**Pull request by skrollme**
## Description

Adds some basic docker commands to help with recurring tasks
- `docker system prune -f`: clears old stuff
- `docker ps`: lists running containers
- `docker images`: lists images

## Type of change

- [x] New script command

## Screenshot

Bildschirmfoto 2021-06-04 um 16 25 25

## Dependencies / Requirements

- Install Docker for Mac: https://docs.docker.com/docker-for-mac/install/

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/docker/docker-images.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Dependency: This script requires `docker for mac` to be installed: https://docs.docker.com/docker-for-mac/install/
+#
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title docker images
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-images.sh:7]:
```suggestion
# @raycast.title List Images
```

File: commands/developer-utils/docker/docker-images.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# @raycast.mode fullOutput
+#
+# Optional parameters:
+# @raycast.icon images/docker.png
+# @raycast.packageName Developer Utilities
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-images.sh:12]:
```suggestion
# @raycast.packageName Docker
```

File: commands/developer-utils/docker/docker-ps.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Dependency: This script requires `docker for mac` to be installed: https://docs.docker.com/docker-for-mac/install/
+#
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title docker ps
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-ps.sh:7]:
```suggestion
# @raycast.title List Containers
```

File: commands/developer-utils/docker/docker-ps.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# @raycast.mode fullOutput
+#
+# Optional parameters:
+# @raycast.icon images/docker.png
+# @raycast.packageName Developer Utilities
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-ps.sh:12]:
```suggestion
# @raycast.packageName Docker
```

File: commands/developer-utils/docker/docker-ps.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+#
+# @raycast.description Runs docker ps
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-ps.sh:14]:
```suggestion
# @raycast.description List containers in Docker
```

File: commands/developer-utils/docker/docker-images.sh [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+#
+# @raycast.description Runs docker images
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-images.sh:14]:
```suggestion
# @raycast.description List images in Docker
```

File: commands/developer-utils/docker/docker-system-prune.sh [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+# Dependency: This script requires `docker for mac` to be installed: https://docs.docker.com/docker-for-mac/install/
+#
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title docker system prune
+# @raycast.mode inline
+#
+# Optional parameters:
+# @raycast.icon images/docker.png
+# @raycast.packageName Developer Utilities
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-system-prune.sh:12]:
```suggestion
# @raycast.packageName Docker
```

File: commands/developer-utils/docker/docker-system-prune.sh [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+#
+# @raycast.description Runs docker system prune -f
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-system-prune.sh:14]:
```suggestion
# @raycast.description Remove unused data (system prune)
```

File: commands/developer-utils/docker/docker-system-prune.sh [hunk 9]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Dependency: This script requires `docker for mac` to be installed: https://docs.docker.com/docker-for-mac/install/
+#
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title docker system prune
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-system-prune.sh:7]:
```suggestion
# @raycast.title System Prune
```

File: commands/developer-utils/docker/docker-system-prune.sh [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+# @raycast.mode inline
```
Comment (unnamedd) [inline commands/developer-utils/docker/docker-system-prune.sh:8]:
```suggestion
# @raycast.mode silent
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hello @skrollme, I made some changes here to make your tags conformant to what indicate the guideline.

Let us know when you finish to a next iteration.

Thanks for your first contribution and welcome aboard!

**Comment by skrollme** [comment]
Thanks for the suggestions, @unnamedd . I applied all but the last suggestions from you because the `inline` was on purpose, to see the _"Total reclaimed space: xy Byte"_ output. But if this does not fit your general look and feel I'm totally fine with switching to `silent`. Just let me know and I apply this as well.

**Comment by unnamedd** [comment]
> Thanks for the suggestions, @unnamedd . I applied all but the last suggestions from you because the `inline` was on purpose, to see the _"Total reclaimed space: xy Byte"_ output. But if this does not fit your general look and feel I'm totally fine with switching to `silent`. Just let me know and I apply this as well.

Ahhh okay, I didn’t know! 
The only thing in the use of the mode `inline` is that you will need to use a `refreshTime` as well. 

Check this document here: https://github.com/raycast/script-commands/blob/master/documentation/OUTPUTMODES.md

If you need anything, please let me know here or on Slack. (@ tholanda)

**Comment by skrollme** [comment]
Ah, ok. Thanks for clarification. Then `compact` is the right output mode here :D I just updated the MR. Thanks for your support and the great software you created for us.

**Comment by unnamedd** [comment]
No problem man, it’s a pleasure to help and clarify your questions, regarding Raycast be an amazing software, please, I’d encourage you to use the Feedback form and send these kind words for the team who is building that amazing software. I’m not part of the team, just a community member.

**Comment by skrollme** [comment]
Good to know, but you help keeping it great. 😉

**Pull request merged**